### DISCUSSION TITLE
Can't get SysOp account to Show on Last Calls / Online

### DISCUSSION THREAD

**Discussion by fc757** [NuSkooler/enigma-bbs#421] [General]
I would like my sysop account to show up as online and in the last calls, but I'm not having any luck getting it work.

This is what I've tried in my menu hjson file:

                fullLoginSequenceLastCallers: {
                        desc: Last Callers
                        module: last_callers
                        art: LASTCALL
                        config: {
                                pause: true
                                font: cp437
                                sysop: {
                                        hide: never
                                        collapse: 4 hour
                                }
                        }
                        next: fullLoginSequenceWhosOnline
                        }

passed on info from https://nuskooler.github.io/enigma-bbs/modding/last-callers.html

I've also tried taking my sysop account out of the groups sysops based on this:

https://github.com/NuSkooler/enigma-bbs/issues/91

I've also tried the config option stated here:

https://github.com/NuSkooler/enigma-bbs/issues/88

Any idea how I can my sysop account show back up again and not be stealth?

Thanks for any help you can provide.

**Comment by NuSkooler** [comment]
@fc757 I just tested it, and it's working as per the docs. I'll push a update to clarify that `hide` is a boolean. By default the system will **show** +ops.

Here is my entry from the main menu in a vanilla install (with the 'sysop' section added by me):
```hjson
mainMenuLastCallers: {
    desc: Last Callers
    module: last_callers
    art: LASTCALL
    config: {
        pause: true
        sysop: {
            // default:true, or you can set to true|false
            hide: false
            collapse: 2 hours
        }
    }
}
```

**Reply by fc757** [reply]
Thanks for checking. I corrected the time from hour to hours, but unfortunately, it's still not working. :(

Not sure what is up. While poking around I noticed that oputil.js shows the last login for the account being back in 2019. Maybe for some reason the database isn't getting update for my admin account. Other users are showing up properly. I also made a new account today and it is showing up too.

۲enigma@cetialphafive۲ Sun May 08 01:38:15pm
~/ ./oputil.js user info fc
User information:
Username     : fC (root/SysOp)
Real name    : fC
ID           : 1
Status       : active
Groups       : users, sysop
Theme ID     : planet_afr0
Created      : 2018-07-01T06:09:34-04:00
Last login   : 2019-09-08T14:53:17-04:00
Login count  : 932
Email        : fc@cetialphafive.com
Location     : PTown
Affiliations : UNiON

Thanks for checking again though. I'll keep poking around. Maybe I need to start over. This install has been around for a while and upgraded a number of times over the years. I probably dorked something up.

**Reply by fc757** [reply]
As a test I moved out the user.sqlite3 file and had it recreate. It is working with the SysOp account now like it should. I'll take a deeper look at the sqlite3 file and see if I can find the issue. If not, well... all good things, etc. Fresh start.

~/ ./oputil.js user info fc
User information:
Username     : fC (root/SysOp)
Real name    : fC
ID           : 1
Status       : active
Groups       : users, sysops
Theme ID     : planet_afr0
Created      : 2022-05-08T13:42:10-04:00
Last login   : 2022-05-08T13:42:10-04:00
Login count  : 1
Email        : fc@planetafr0.org
Location     : PTown
Affiliations : UNiON

**Reply by NuSkooler** [reply]
@fc757 From your older install, check the `UPGRADE.md` and make sure you caught the DB migrations in your upgrades. I know you've used this for a while, I wonder if you need this one:
```
sqlite3 db/system.db DROP TABLE user_event_log; # then restart
```

**Comment by fc757** [comment]
OK. I figured it out. I did not drop that table, but unfortunately that wasn't the issue. The issue was my account had google 2FA enabled and wasn't working. I was logging in with out and I guess that caused the stuff not to update. All is well now.

Thanks for your help and pushing me in the right direction.
